src/command_modules/azure-cli-storage/tests/test_storage_account_scenarios.py

```diff
@@ -80,23 +79,16 @@ def test_logging_operations(self, resource_group, storage_account):
 
     @ResourceGroupPreparer()
     @StorageAccountPreparer()
-    def test_metrics_operations(self, resource_group, storage_account):
-        connection_string = self.cmd(
-            'storage account show-connection-string -g {} -n {} -otsv'
-            .format(resource_group, storage_account)).output
+    def test_metrics_operations(self, resource_group, storage_account_info):
+        self.storage_cmd('storage metrics show', storage_account_info) \
+            .assert_with_checks(JMESPathCheck('file.hour.enabled', True),
+                                JMESPathCheck('file.minute.enabled', False))
 
-        self.cmd('storage metrics show --connection-string "{}"'.format(connection_string), checks=[
-            JMESPathCheck('file.hour.enabled', True),
-            JMESPathCheck('file.minute.enabled', False),
-        ])
-
-        self.cmd('storage metrics update --services f --hour false --retention 1 '
-                 '--connection-string "{}"'.format(connection_string))
+        self.storage_cmd('storage metrics update --services f --hour false --retention 1 ', storage_account_info)
 
-        self.cmd('storage metrics show --connection-string "{}"'.format(connection_string), checks=[
-            JMESPathCheck('file.hour.enabled', False),
-            JMESPathCheck('file.minute.enabled', False),
-        ])
+        self.storage_cmd('storage metrics show', storage_account_info) \
```

Comment:
Should this test be expanded to test the logic of setting different retention periods for hourly and by-minute metrics?  Or is that not part of this PR?